### ISSUE TITLE
Real Trouble Through-Hole Plating

### ISSUE THREAD

**Issue by abeyene** [bespoke-silicon-group/bsg_motherboards#5]
Hi,

I've received a query from my PCB manufacturer about whether a particular through-hole on Real Trouble is plated or not. A copper pad exists in the copper layer but a table description for the symbol describes the hole as non-plated (see the below photo). Which is correct? Thanks.

![RealTroubleThroughHole](https://user-images.githubusercontent.com/36420176/155236727-0bfee026-b0ba-4a93-9cae-78b065596deb.png)

**Comment by taylor-bsg** [comment]
@gaozihou

**Comment by gaozihou** [comment]
@abeyene Thanks for your question! 
This should be a **plated** through hole (PTH), so the **table is incorrect**..
Sorry for the inconvenience caused!

**Comment by cmcjames** [comment]
Paul-

  Isn’t that the FMC connector’s keying hole?

    The SAMTEC datasheet’s recommended PCB layout section for an FMC connector shows the keying hole as 50 [1.27] NPTH.
    In the VITA 57.1 standard “Rule 3.25: All keying holes shall remain electrically isolated from the mezzanine module’s circuit ground.”

  I assume it doesn’t matter as the key is likely some kind of plastic.

  What is the reasoning for it being a PTH?  Am I missing something?

-james

**Comment by gaozihou** [comment]
Thanks @cmcjames!

Yes you are right, Samtec datasheet suggests 1.27 NPTH for those keying holes, so it is totally fine to go with NPTH.

I suggested PTH because

**Comment by gaozihou** [comment]
Thanks @cmcjames!

Yes you are right, Samtec datasheet suggests 1.27 NPTH for those keying holes, so it is totally fine to go with NPTH.

I suggested PTH because
- We used PTH in last fabrication and it worked..
- It matches the plated keying hole on the other side of the FMC..
- For most manufactures the 1.27 diameter is finished size, so the FMC key will fit in the plated hole

Again sorry for the confusions caused!
